### DISCUSSION TITLE
I want to help with translation Chinese Traditional.

### DISCUSSION THREAD

**Discussion by DAS-WIN** [Chia-Network/chia-blockchain#14780] [Q&A]
I want to help with translation work, but this project has been blocked. How can I still do translation work? Please help me format this sentence into English.

The translation website (crowdin.com) displays the message "The project you are trying to translate is currently suspended."
![image](https://user-images.githubusercontent.com/50190791/223743829-f67d6d3c-f35f-46d1-91cd-8d31ace92c06.png)

**Comment by keliew** [comment]
That page is deprecated. It's now at https://chia.crowdin.com/chia-blockchain-gui

The documentation should be updated accordingly: https://docs.chia.net/faq?_highlight=translation#how-can-i-contribute
